.circleci/config.yml

```diff
@@ -643,22 +690,17 @@ jobs:
       - checkout
       - check-changed:
           patterns: specs/(.*)\.md$
-      # TODO remove me after ci builder updated
       - run:
           name: Install pnpm package manager
           command: |
             npm i pnpm --global
-      # TODO remove me after ci builder updated
-      # A github dep clones-with-immutable-args is installed via github
-      # packages installed via npm via github automatically run postpack scripts
-      # their postpack script happens to use yarn so we need it here
       - run:
           name: Install yarn package manager
           command: |
```

Comment:
We now install solidity deps with git submodules so I don't think we need yarn here unless for a diff reason from the removed comment

Reply:
Oh yea likely fixed now